.github/actions/multi-functest/action.yml

```diff
@@ -107,7 +109,7 @@ runs:
           nix-verbose: ${{ inputs.nix-verbose }}
           gh_token: ${{ inputs.gh_token }}
           custom_shell: ${{ inputs.custom_shell }}
-          cflags: "${{ inputs.cflags }}"
+          cflags: "${{ inputs.cflags }} -DMLD_FORCE_PPC64LE"
```

Comment:
There is no MLD_FORCE_PPC64LE flag yet. Add it?

Reply:
Done